pull/github.go

```diff
@@ -982,6 +982,9 @@ type v4PullRequestReview struct {
 	State       string
 	Body        string
 	SubmittedAt time.Time
+	Commit      struct {
+		OID string
```

Comment:
Is `OID` `Object ID`? It feels a bit funky that the graphql api doesn't just call this `sha`.